apps/src/tutorialExplorer/responsive.jsx

```diff
@@ -0,0 +1,96 @@
+/* A very simple responsive layout system.
+ */
+
+/**
+ * Gets the container width.
+ * Returns either a number (e.g. 1170) or a string (e.g. "97%").
+ */
+
+function getResponsiveContainerWidth() {
+  const windowWidth = $(window).width();
+
+  if (windowWidth >= 1200) {
+    return 1170;
+  } else {
+    return "97%";
+  }
+}
+
+/* Window widths that are the starting points for each width category. */
+const responsiveWindowWidth = {
+  lg: 1024,
+  md: 820,
+  sm: 650,
+  xs: 0
+};
```

Comment:
Any reason not to use full words instead of abbreviations?  Might also make sense to create an enum instead of using strings, i.e.:

```
// makeEnum comes from apps/src/utils
const ResponsiveSize = makeEnum('Large', 'Medium', 'Small', 'ExtraSmall');

const responsiveWindowWidth = {
  [ResponsiveSize.Large]: 1024,
  [ResponsiveSize.Medium]: 820
}
```